.github/workflows/check-push.yml

```diff
@@ -0,0 +1,64 @@
+name: Develop Workflow with .Net
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+permissions: read-all      
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      
+      - name: Build using .Net
+        run: dotnet build
+
+  semantic_pull_request:
```

Comment:
A file related for push events should not have jobs related to PRs events.

Reply:
done